nav2_bringup/bringup/launch/navigation_launch.py

```diff
@@ -53,6 +54,7 @@ def generate_launch_description():
     param_substitutions = {
         'use_sim_time': use_sim_time,
         'default_bt_xml_filename': default_bt_xml_filename,
+        'bt_xml_filename': waypoint_follower_bt_xml_filename,
```

Comment:
If you now have 2 different BT files, you should make them both clear as to which is which. Though do you really need to make a launch argument out of this?

Reply:
No.
I'll remove it.

Reply:
I've remove it.